### ISSUE TITLE
Getting SyntaxError when running basic command

### ISSUE THREAD

**Issue by avindra** [yarnpkg/yarn#613]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
```
$ yarn
yarn install v0.15.1
error SyntaxError: Unknown token 1:0
    at Parser.unexpected (/app/node/lib/node_modules/yarnpkg/lib/lockfile/parse.js:218:11)
    at Parser.parse (/app/node/lib/node_modules/yarnpkg/lib/lockfile/parse.js:323:14)
    at exports.default (/app/node/lib/node_modules/yarnpkg/lib/lockfile/parse.js:13:17)
    at /app/node/lib/node_modules/yarnpkg/lib/registries/yarn-registry.js:105:62
    at next (native)
    at step (/app/node/lib/node_modules/yarnpkg/node_modules/babel-runtime/helpers/asyncToGenerator.js:17:30)
    at /app/node/lib/node_modules/yarnpkg/node_modules/babel-runtime/helpers/asyncToGenerator.js:28:20
info Visit http://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**If the current behavior is a bug, please provide the steps to reproduce.**
Just run `yarn`

**What is the expected behavior?**
It should install all dependencies.

**Please mention your node.js, yarn and operating system version.**
```
yarn:  v0.15.1
node: v6.5.0
OS:  SLES 11 SP3
```

**Comment by kittens** [comment]
Do you have a `yarn.lock` file? If so what's in it? Looks like we're failing to parse it, and should only be doing that if there's a lockfile.

**Comment by maniator** [comment]
Same issue with me. I have no yarn.lock file.

**Comment by avindra** [comment]
No `yarn.lock` file:

``
$ ls -l yarn.lock
ls: cannot access yarn.lock: No such file or directory
```

I also deleted my `npm-shrinkwrap.json` for good measure, but that didn't help

**Comment by avindra** [comment]
Updated to node 6.7.0 and still encountering the same issue. 

Strangely enough, openSUSE Tumbleweed (my desktop at home) works perfectly fine (running 6.7.0 there as well). It's just the linux machines at my work that have this problem. @maniator : What OS are you trying from?

@kittens : if you can provide some suggestions, I can try to help debugging. Happy to assist

**Comment by avindra** [comment]
Just discovered it's related to `~/.yarnrc`, not the lock file. The `strict-ssl` property (which I had defined in `.yarnrc`) is not supported yet it seems, which is why this error is being thrown.

**Comment by avindra** [comment]
Closing this in favor of further discussion on https://github.com/yarnpkg/website/issues/128 and https://github.com/yarnpkg/yarn/issues/606

**Comment by avindra** [comment]
For the record, my issue is resolved by `0.16` :raised_hands: :tada:
